kratos/containers/pointer_hash_map_set.h

```diff
@@ -178,12 +178,12 @@ class PointerHashMapSet
 
     TDataType& operator[](const key_type& Key)
     {
-		return *(mData[Key].second);
+		return *(mData.find(Key)->second);
```

Comment:
👍

Reply:
shouldn't be `return *(mData[Key]);`?

Reply:
OK, discussing with @RiccardoRossi, the find is better because don't create it if is not there. So it will safely give you a segmentation fault in parallel. Nevertheless, I would add a KRATOS_DEBUG check.

Reply:
Done